src/index.html

```diff
@@ -1,13 +1,321 @@
+                  alt="How the Phone is hold up on the Bike"
+                  class="topic__image"
+                >
+              </a>
+
+              <h3 class="topic__title">Auto Unlock</h3>
+
+              <p class="topic__description">
+                The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.
+              </p>
+            </article>
+
+            <article class="details__topic topic">
+              <a href="#" class="topic__link topic__link--wide">
+                <img
+                  src="./images/details/range-lights-wide.png"
+                  alt="Light on Bike behind"
+                  class="topic__image"
+                >
+              </a>
+
+              <a href="#" class="topic__link topic__link--square">
+                <img
+                  src="./images/details/range-lights-square.png"
+                  alt="Light on Bike toward"
+                  class="topic__image"
+                >
+              </a>
+
+              <h3 class="topic__title">Range & Integrated lights</h3>
+
+              <p class="topic__description">
+                The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+              </p>
+            </article>
+
+            <article class="details__topic topic">
+              <a href="#" class="topic__link topic__link--square">
+                <img
+                src="./images/details/hydraulic-lightweight-square.png"
+                alt="Bike brake handle"
+                class="topic__image"
+                >
+              </a>
+
+              <a href="#" class="topic__link topic__link--wide">
+                <img
+                  src="./images/details/hydraulic-lightweight-wide.png"
+                  alt="Bike footrest"
+                  class="topic__image"
+                >
+              </a>
+
+              <h3 class="topic__title">Auto Unlock</h3>
+
+              <p class="topic__description">
+                The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.
+              </p>
+            </article>
+
+            <a href="#" class="details__button button">Explore</a>
+          </div>
+        </section>
+
+        <section class=" main-content__contact contact" id="contact">
+          <h2 class="contact__title">Contact us</h2>
+
+          <div class="contact__content">
+            <form
+              action="/"
+              method="get"
+              class="contact__form form"
+              onsubmit="this.reset(); return false;"
+            >
+              <input
+                type="text"
+                placeholder="Name"
+                class="form__text"
+                autocomplete="given-name"
+              >
```

Comment:
Add the `name` attribute to all of the inputs but they should be unique at all inputs